src/app/zap-templates/zcl/data-model/chip/electrical-power-measurement-cluster.xml

```diff
@@ -0,0 +1,98 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2023 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<configurator>
+  <domain name="CHIP"/>
+  <bitmap name="Feature" type="bitmap32">
+    <cluster code="0x0090"/>
+    <field name="DirectCurrent" mask="0x01"/>
+    <field name="AlternatingCurrent" mask="0x02"/>
+    <field name="PolyphasePower" mask="0x04"/>
+    <field name="Harmonics" mask="0x08"/>
+    <field name="PowerQuality" mask="0x10"/>
+  </bitmap>
+  <cluster apiMaturity="provisional">
+    <name>Electrical Power Measurement</name>
+    <domain>Measurement &amp; Sensing</domain>
+    <code>0x0090</code>
+    <define>ELECTRICAL_POWER_MEASUREMENT_CLUSTER</define>
+    <client init="false" tick="false">true</client>
+    <server init="false" tick="false">true</server>
+    <globalAttribute side="either" code="0xFFFD" value="1"/>
+    <description>This cluster provides a mechanism for querying data about electrical power as measured by the server.</description>
+    <!--Attributes-->
+    <attribute code="0x0000" side="server" define="POWER_MODE" type="PowerModeEnum" min="0x00" max="0x02">PowerMode</attribute>
+    <attribute code="0x0001" side="server" define="NUMBER_OF_MEASUREMENT_TYPES" type="int8u" min="1">NumberOfMeasurementTypes</attribute>
+    <attribute code="0x0002" side="server" define="ACCURACY" type="array" entryType="MeasurementAccuracyStruct" minLength="1">Accuracy</attribute>
+    <attribute code="0x0003" side="server" define="RANGES" type="array" optional="true" entryType="MeasurementRangeStruct" minLength="0">Ranges</attribute>
+    <attribute code="0x0004" side="server" define="VOLTAGE" type="voltage_mv" isNullable="true" min="0xC000000000000000" max="0x4000000000000000" optional="true" default="0x8000000000000000">Voltage</attribute>
```

Comment:
I'm not sure you need these defaults for null, the DataModel::Nullable type has a flag to say if it HasValue() so the fact it `isNullable="true"` means that there doesn't need to be a reserved value, but please check with other experts.